lib/collection/src/lookup/mod.rs

```diff
@@ -22,12 +22,23 @@ pub struct WithLookup {
     pub collection_name: String,
 
     /// Options for specifying which payload to include (or not)
+    #[serde(default = "default_with_payload")]
     pub with_payload: Option<WithPayloadInterface>,
 
     /// Options for specifying which vectors to include (or not)
+    #[serde(alias = "with_vector")]
+    #[serde(default = "default_with_vectors")]
     pub with_vectors: Option<WithVector>,
 }
 
+const fn default_with_vectors() -> Option<WithVector> {
```

Comment:
I think other retrieval methods are not returning any stored info by default. 
https://qdrant.tech/documentation/concepts/search/#payload-and-vector-in-the-result

Reply:
+1, payload should be included, but vectors are too big and to useless in regular scenario